packages/SwingSet/docs/timer.md

```diff
@@ -67,18 +67,23 @@ A single application might have multiple sources of time, which would require th
 The `timerService` object can be distributed to other vats as necessary.
 
 ```js
-  // for this example, assume poll() provides seconds-since-epoch
+  import { TimeMath } from '@agoric/time';
+
+  // for this example, assume timerService provides seconds-since-epoch
+  const timerBrand = await E(timerService).getTimerBrand();
+  const toTS = ts => TimeMath.coerceTimestampRecord(ts, timerBrand);
+  const toRT = value => TimeMath.coerceRelativeTimeRecord(value, timerBrand);
 
   const now = await E(timerService).getCurrentTimestamp();
 
   // simple one-shot Promise-based relative delay
-  const p1 = E(timerService).delay(30n); // fires 30 seconds from now
+  const p1 = E(timerService).delay(toTS(30n)); // fires 30 seconds from now
```

Comment:
`delay()` takes a relative timestamp, so that needs `toRT`

When I picked `coerceTimestampRecord`, I thought the awkwardly-long name would be ok because clients would almost never need to use it: you get timestamps from the service, you send them back to the service, you get offsets from `now` with `TimeMath.addAbsRel`, and you never create one from whole cloth.

But this example makes me think that the relative-time flavor (`coerceRelativeTimeRecord`) *is* a user-facing API, and that having a long name for it is kind of annoying. Assuming clients know the scale of the time in question, then it's ok for code to start with a plain `Number` when building a relative time record. So we'd like a short way to express the `toRT` in this example.

That said, it's not unusably long. And `coerce` isn't entirely wrong, that's definitely what it's doing.

So, let's pay attention to how this API feels to use, and let's consider re-introducing `TimeMath.toRel` in the future. It wouldn't change anything about the types, just the relative visibility/ergonomic considerations of this particular function.